pkg/controller/resourceclaim/metrics/metrics.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// ResourceClaimSubsystem - subsystem name used for ResourceClaim creation
+const ResourceClaimSubsystem = "resource_claim_controller"
```

Comment:
Since this is referring to the ResourceClaim resource, I think `resourceclaim_controller` would be a better name for the subsystem.

Reply:
Makes sense, changed.